gostream/stream.go

```diff
@@ -31,6 +33,7 @@ type Stream interface {
 
 	// Start starts processing frames.
 	Start()
+	WriteRTP(pkt *rtp.Packet) error
```

Comment:
Was this a thing that we were concerned was going to change the camera API?

Reply:
no:

This is:
https://github.com/viamrobotics/rdk/pull/3723/files#diff-4eab9b93978c09650defc23c4274a4c77d87cfdea7775d86dcf8c838ec207a01R85-R98